src/components/switch-network-message.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import t from "prop-types";
+import styled from "styled-components/macro";
+import { Typography } from "antd";
+import { chainIdToNetworkName } from "../helpers/networks";
+
+export default function SwitchNetworkMessage({ title, wantedChainId }) {
+  return (
+    <StyledSwitchNetworkMessage>
```

Comment:
Maybe we can add a button here to switch the network? Would improve UX.

Reply:
This component is used is some places where there's already a button for that when the action is available. It would be too repetitive.
Perhaps I made a mistake when I extracted it to be a general component. Will refactor it.